iothub_client/src/iothubtransport_mqtt_common.c

```diff
@@ -2626,26 +2626,12 @@ static int SendMqttConnectMsg(PMQTTTRANSPORT_HANDLE_DATA transport_data)
     }
     else if (cred_type == IOTHUB_CREDENTIAL_TYPE_SAS_TOKEN)
     {
-        SAS_TOKEN_STATUS token_status = IoTHubClient_Auth_Is_SasToken_Valid(transport_data->authorization_module);
-        if (token_status == SAS_TOKEN_STATUS_INVALID)
-        {
-            transport_data->transport_callbacks.connection_status_cb(IOTHUB_CLIENT_CONNECTION_UNAUTHENTICATED, IOTHUB_CLIENT_CONNECTION_EXPIRED_SAS_TOKEN, transport_data->transport_ctx);
-            result = MU_FAILURE;
-        }
-        else if (token_status == SAS_TOKEN_STATUS_FAILED)
+        sasToken = IoTHubClient_Auth_Get_SasToken(transport_data->authorization_module, NULL, 0, NULL);
+        if (sasToken == NULL)
         {
-            transport_data->transport_callbacks.connection_status_cb(IOTHUB_CLIENT_CONNECTION_UNAUTHENTICATED, IOTHUB_CLIENT_CONNECTION_BAD_CREDENTIAL, transport_data->transport_ctx);
+            LogError("failure getting sas Token.");
```

Comment:
Same here

Reply:
delete line 2637???

Reply:
Sorry no same as the previous change for the log error. Capitalization

Reply:
Nit: changing log may have adverse effects on automatic log scanners.
I don't think we've ever guaranteed log backward compatibility, just something to consider, especially given our design that won't report different error codes (i.e. `MU_FAILURE` is set to indicate a line-number and will change in value with this modification).